common/src/main/java/com/wynntils/models/players/type/GuildInfo.java

```diff
@@ -9,9 +9,14 @@
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParseException;
+import com.wynntils.core.WynntilsMod;
 import java.lang.reflect.Type;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Date;
```

Comment:
`java.util.Date` is severely outdated. You should only use classes and functionality from the new `java.time` package.